ext/mysqli/tests/mysqli_change_user_new.phpt

```diff
@@ -11,6 +11,9 @@ if (!$link = my_mysqli_connect($host, $user, $passwd, $db, $port, $socket))
 
 if (mysqli_get_server_version($link) < 50600)
```

Comment:
is the condition correct, comment says something different?

Reply:
The condition is correct, but the premise for the condition might be flawed. See comment below.